Assets/MixedRealityToolkit/Providers/Hands/BaseHand.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.MixedReality.Toolkit.Input
     public abstract class BaseHand : BaseController, IMixedRealityHand
     {
         // Hand ray
```

Comment:
nit: while you are here, can you remove the hand ray comment? It doesn't provide any value.